packages/ui-lib/src/Form/molecules/ButtonsBundle.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import styled from 'styled-components';
+import ButtonWithIcon from '../atoms/ButtonWithIcon';
+import Button from '../atoms/Button';
+import { IButtonProps } from '..';
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  button {
+    text-wrap: nowrap;
+  }
+`;
+
+export type IButtonType = 'default' | 'icon-button'
+
+export interface IBButton extends IButtonProps {
+  id: string
```

Comment:
I believe `ID` here should be optional rather than required.  

Looking at our components, it's not a mandatory prop for anything moving through the different levels. If I've missed something though, please let me know.

Reply:
The reason is to assign the key and  prevent the warning that React gives when you are rendering a list of items. 
https://react.dev/learn/rendering-lists. 

I want to delegate the control of the id's to the Dev user in case they need it.

Reply:
Thank you for the link. I've got a few extra thoughts on this that I'd like to chat about quickly, I'll send you a message in Slack about it - likely though there won't need to be any other changes on here.

Reply:
Updated with key={id || `button-stack-${index}`} as we agreed in chat.